www/runtest.php

```diff
@@ -210,7 +210,7 @@ function DealWithMagicQuotes(&$arr) {
               $test['lighthouse'] = $req_lighthouse;
             $test['lighthouseTrace'] = isset($_REQUEST['lighthouseTrace']) && $_REQUEST['lighthouseTrace'] ? 1 : 0;
             $test['lighthouseThrottle'] = isset($_REQUEST['lighthouseThrottle']) && $_REQUEST['lighthouseThrottle'] ? 1 : GetSetting('lighthouseThrottle', 0);
-            $test['heroElementTimes'] = isset($_REQUEST['heroElementTimes']) && $_REQUEST['heroElementTimes'] ? 1 : 0;
+            $test['heroElementTimes'] = isset($_REQUEST['heroElementTimes']) ? (int) $_REQUEST['heroElementTimes'] : 1;

```

Comment:
Can you change this instead to operate like the "lighthouseThrottle" line above and pull the default value from the settings file? Also add a sample to settings.ini.sample